deploy.sh

```diff
@@ -2,43 +2,38 @@
 
 push_helm_charts() {
 	PACKAGE=$1
-	CHANGED=0
-	VERSION=${ENV_CSI_IMAGE_VERSION//v}  # Set version (without v prefix)
+	VERSION=${ENV_CSI_IMAGE_VERSION//v/} # Set version (without v prefix)
 
-  # Always run when version is canary, when versioned only when the package doesn't exist yet
-	if [ ! -f "tmp/csi-charts/docs/$PACKAGE/ceph-csi-$PACKAGE-$VERSION.tgz" ] && [ -z "$VERSION" ]; then
-		CHANGED=1
+	# update information in Chart.yaml if the branch is not master
+	if [ "$TRAVIS_BRANCH" != "master" ]; then
```

Comment:
I think this does not work since https://github.com/ceph/ceph-csi/blob/ec1002c511acbffb4490db06b773c765465bb7c3/deploy.sh#L36 exists the script when the branch isn't master.

Reply:
check here in the release branch https://github.com/ceph/ceph-csi/blob/release-v1.2.0/deploy.sh#L38